enterprise-suite/tests/smoke_ingress

```diff
@@ -6,6 +6,8 @@
 
 source smokecommon
 
+if [[ $context == "GKE" ]]; then exit 0; fi
```

Comment:
I'm confused.  The rest of these changes seem to be related to GKE ingress things but both ingress tests are disabled if we're using GKE?

Reply:
yeah I tend to agree that this seems odd.  Why wouldn't we run smoke_ingress etc in gke?